Задача 1.

Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800]. Найдите ее среднее значение(500) и дисперсию.

In [1]:
import numpy as np

In [2]:
A = [200,800]
A_mean = (200+800)/2
A_mean

500.0

In [3]:
D_A = ((800-200)**2)/12
D_A

30000.0

Задача 2.

О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2. Можно ли найти правую границу величины B и ее среднее значение (0.775) зная, что левая граница равна 0.5? Если да, найдите ее.

In [4]:
d = (-1)**2-4*1*(-2.15)
d

9.6

In [5]:
x_1 = ((-1)+np.sqrt(9.6))/2
x_1

1.0491933384829668

Овет: правая граница величины B = 1.05

In [6]:
x_2 = ((-1)-np.sqrt(9.6))/2
x_2

-2.049193338482967

In [7]:
M_B = (0.5+1.05)/2
M_B

0.775

Ответ: среднее значение В = 0.775

Задача 3.

Непрерывная случайная величина X распределена нормально и задана плотностью распределения f(x) = (1 / (4 * sqrt(2pi))) * exp((-(x+2)**2) / 32) Найдите: а). M(X) б). D(X) в). std(X) (среднее квадратичное отклонение)

а). M(X) = -2
б). D(X) = 16
в). std(X) (среднее квадратичное отклонение) = 4

Задача 4.

Рост взрослого населения города X имеет нормальное распределение. Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см. Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост: а). больше 182 см б). больше 190 см в). от 166 см до 190 см г). от 166 см до 182 см д). от 158 см до 190 см е). не выше 150 см или не ниже 190 см ж). не выше 150 см или не ниже 198 см з). ниже 166 см.

In [8]:
mu = 174
sigma = 8

In [9]:
z_a = (182-mu)/sigma
z_a

1.0

In [10]:
1-0.8413

0.15869999999999995

Ответ: больше 182 см (0.1587)

In [11]:
z_b = (190-mu)/sigma
z_b

2.0

In [12]:
1-0.9772

0.022800000000000042

Ответ: больше 190 см (0.0228)

In [13]:
z_v1 = (166-mu)/sigma
z_v1

-1.0

In [14]:
0.9772-0.1587

0.8185

Ответ: от 166 см до 190 см (0.8185)

In [15]:
z_g = z_a-z_v1
z_g

2.0

In [16]:
0.8413-0.1587

0.6826000000000001

Ответ: от 166 см до 182 см (0.6826)

In [17]:
z_d = (158-mu)/sigma
z_d

-2.0

In [18]:
0.9772-0.0228

0.9543999999999999

Ответ: от 158 см до 190 см (0.9544)

In [19]:
z_e1 = (150-mu)/sigma
z_e1

-3.0

In [20]:
z_e=0.0013+0.0228
z_e

0.0241

Ответ: не выше 150 см или не ниже 190 см (0.0241)

In [21]:
z_yo1 = (198-mu)/sigma
z_yo1

3.0

In [22]:
z_yo = 1-0.9986 +0.0013
z_yo

0.0026999999999999568

Ответ: ниже 166 см. (0.1587)

Задача 5.

На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [23]:
sigma_delta = (190-178)/np.sqrt(25)
sigma_delta

2.4

Ответ: 2.4